logstash-filter-kubernetes_metadata.gemspec

```diff
@@ -0,0 +1,26 @@
+Gem::Specification.new do |s|
+  s.name          = 'logstash-filter-kubernetes_metadata'
+  s.version       = '1.0.2'
+  s.licenses      = ['AGPL-3.0']
```

Comment:
I generally use AGPL-3 on most of my code but don't mind flipping it to Apache 2.0. Updated.